doc/examples/filters/plot_denoise.py

```diff
@@ -38,7 +38,7 @@
 noisy = astro + 0.6 * astro.std() * np.random.random(astro.shape)
```

Comment:
@sciunto resolved. Instead of uniform noise I use Gaussian nose through `random_noise` with the same standard deviation.